docs/advanced/custom_operators.ipynb

```diff
@@ -6,7 +6,7 @@
    "id": "44701f85-3e69-4e1c-b017-54a843297568",
```

Comment:
<p>now I am confused, why this is deriving from AbstractObservable and not AbstractOperator? (the text still says AbstractOperator btw) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1605/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>